### PR TITLE
Override WebElement.isSelected for EditorTab class

### PR DESCRIPTION
**Pull request by mlorinc**
Solves #606 

Signed-off-by: Marian Lorinc <mlorinc@redhat.com>

### CODE REVIEW

**Review by ssbarnea — DISMISSED** [review]
LGTM but wait for another review on this one.

File: test/test-project/src/test/editor/editorView-test.ts
Comment (djelinek) [inline test/test-project/src/test/editor/editorView-test.ts:167]: don't you think that this could be part of EditorView? I think it is pitty to have these kind of methods located only in test classes...

File: test/test-project/src/test/editor/editorView-test.ts
Comment (djelinek) [inline test/test-project/src/test/editor/editorView-test.ts:193]: I think that it is quite too much complex workaround ale again it is pity to have this kind of logic directly in test class...
the new file creation could be done easier in first iteration..

but I like this kind of solid and robust approach.. we should discuss how to reuse this code and add another like that one..

AI: discuss some kind of experimental FileSystem components which could be shipped as part of extester

**Review by djelinek — CHANGES_REQUESTED** [review]

File: test/test-project/src/test/editor/editorView-test.ts
Comment (mlorinc) [reply test/test-project/src/test/editor/editorView-test.ts]: I have introduced EditorTabNotFound to fulfil this use case.

File: test/test-project/src/test/editor/editorView-test.ts
Comment (mlorinc) [reply test/test-project/src/test/editor/editorView-test.ts]: Extester does not have any click/sendKeys limiters which would be nice to have and make testing more straightforward.

**Review by djelinek — APPROVED** [review]

**Pull request merged**
